maven/plugins/common/src/main/java/com/vmware/pscoe/maven/plugins/AbstractInstallNodeDepsMojo.java

```diff
@@ -80,10 +83,41 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 				.execute(getLog());
         }
 
-        new ProcessExecutor()
-			.name("Dependency installation")
-			.directory(project.getBasedir())
-			.command(deps)
-			.execute(getLog());
+        getLog().info("deps length  " + deps.stream().mapToInt(String:: length).sum());
```

Comment:
Can you also change it to String::length ( without the space )

Reply:
Fixed